web/packages/shared/components/FileTransfer/FileTransferActionBar.tsx

```diff
@@ -34,24 +34,24 @@ export function FileTransferActionBar({
 }: FileTransferActionBarProps) {
   const fileTransferContext = useFileTransferContext();
   const areFileTransferButtonsDisabled =
-    fileTransferContext.openedDialog || !isConnected || !hasAccess;
+    !!fileTransferContext.openedDialog || !isConnected || !hasAccess;
 
   return (
-    <Flex flex="none" alignItems="center" height="24px">
-      <HoverTooltip
-        placement="bottom"
-        tipContent={
-          !hasAccess ? (
-            <Text>
-              You are missing the{' '}
-              <Text bold as="span">
-                ssh_file_copy
-              </Text>{' '}
-              role option.
-            </Text>
-          ) : null
-        }
-      >
+    <HoverTooltip
+      placement="bottom"
+      tipContent={
+        hasAccess ? (
```

Comment:
A leftover from testing that you forgot to undo perhaps? 😏

Reply:
Exactly, thanks for catching this!